airflow/jobs/scheduler_job_runner.py

```diff
@@ -705,12 +705,14 @@ def _process_executor_events(self, session: Session) -> int:
             ti_primary_key_to_try_number_map[ti_key.primary] = ti_key.try_number
 
             self.log.info(
-                "Executor reports execution of %s.%s run_id=%s exited with status %s for try_number %s",
+                "Executor received event with state %s for task instance "
+                "dag_id=%s task_id=%s run_id=%s try_number=%s map_index=%s",
```

Comment:
I wonder if we should just toss ti_key here and let its str do it's thing. Same info.

Reply:
Probably a good idea; it’d be easier if we add another field to the key in the future (like we did map_index; although of course this doesn’t happen at all often anyway)

Reply:
good with that